### DISCUSSION TITLE
第5节课作业(6班)

### DISCUSSION THREAD

**Discussion by zhuyr97** [InternLM/Tutorial#297] [作业]
第5节课作业(6班)

**Comment by zhangzefeng92** [comment]
微信号：捂上眼睛看世界
![图片](https://github.com/InternLM/tutorial/assets/58167242/212b0804-67d0-4cb9-9b16-de9005e3fe50)

**Comment by yokebtc** [comment]
yoke - [https://app.yinxiang.com/fx/1b90335e-3817-4da8-9d33-423cda9f8e7a](https://app.yinxiang.com/fx/1b90335e-3817-4da8-9d33-423cda9f8e7a)

**Comment by RalphGuo** [comment]
guos-https://gitee.com/RalphGs/InternLM-assignment/tree/master/%E8%AF%BE%E7%A8%8B5
